README.md

```diff
@@ -46,12 +46,6 @@ To get a local copy up and running follow these simple example steps.
 - Twitter: [@indigo1987](https://twitter.com/indigo1987)
 - LinkedIn: [LinkedIn](https://linkedin.com/in/david-vera-castillo-001b5756/)
 
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
```

Comment:
- [ ] In your README file, Please add the project name and description and remove unused sections like Prerequisites, Setup, Install, Usage, Run tests, and Deployment. And Kindly add the correct info in `getting started` explaining how to clone the project, install all dependencies, and run the live server.